provider/cmd/pulumi-resource-hcp/schema.json

```diff
@@ -222,13 +229,14 @@
eeringId = \"dev\",\n            PeerVpcId = peerVpc.Id,\n            PeerAccountId = peerVpc.OwnerId,\n            PeerVpcRegion = peerArn.Apply(peerArn =\u003e peerArn.Region),\n        });\n        var main_to_dev = new Hcp.HvnRoute(\"main-to-dev\", new Hcp.HvnRouteArgs\n        {\n            HvnLink = main.SelfLink,\n            HvnRouteId = \"main-to-dev\",\n            DestinationCidr = \"172.31.0.0/16\",\n            TargetLink = dev.SelfLink,\n        });\n        var peerVpcPeeringConnectionAccepter = new Aws.Ec2.VpcPeeringConnectionAccepter(\"peerVpcPeeringConnectionAccepter\", new Aws.Ec2.VpcPeeringConnectionAccepterArgs\n        {\n            VpcPeeringConnectionId = dev.ProviderPeeringId,\n            AutoAccept = true,\n        });\n    }\n\n}\n```\n```go\npackage main\n\nimport (\n\t\"github.com/grapl-security/pulumi-hcp/sdk/go/hcp\"\n\t\"github.com/pulumi/pulumi-aws/sdk/v5/go/aws\"\n\t\"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/ec2\"\n\t\"github.com/pulumi/pulumi/sdk/v3/go/pulumi\"\n)\n\nfunc main() {\n\tpulumi.Run(func(ctx *pulumi.Context) error {\n\t\tmain, err := hcp.NewHvn(ctx, \"main\", \u0026hcp.HvnArgs{\n\t\t\tHvnId:         pulumi.String(\"main-hvn\"),\n\t\t\tCloudProvider: pulumi.String(\"aws\"),\n\t\t\tRegion:        pulumi.String(\"us-west-2\"),\n\t\t\tCidrBlock:     pulumi.String(\"172.25.16.0/20\"),\n\t\t})\n\t\tif err != nil {\n\t\t\treturn err\n\t\t}\n\t\tpeerVpc, err := ec2.NewVpc(ctx, \"peerVpc\", \u0026ec2.VpcArgs{\n\t\t\tCidrBlock: pulumi.String(\"172.31.0.0/16\"),\n\t\t})\n\t\tif err != nil {\n\t\t\treturn err\n\t\t}\n\t\tpeerArn := aws.GetArnOutput(ctx, GetArnOutputArgs{\n\t\t\tArn: peerVpc.Arn,\n\t\t}, nil)\n\t\tdev, err := hcp.NewAwsNetworkPeering(ctx, \"dev\", \u0026hcp.AwsNetworkPeeringArgs{\n\t\t\tHvnId:         main.HvnId,\n\t\t\tPeeringId:     pulumi.String(\"dev\"),\n\t\t\tPeerVpcId:     peerVpc.ID(),\n\t\t\tPeerAccountId: peerVpc.OwnerId,\n\t\t\tPeerVpcRegion: peerArn.ApplyT(func(peerArn GetArnResult) (string, error) {\n\t\t\t\treturn peerArn.Region, nil\n\t\t\t}).(pulumi.StringOutput),\n\t\t})\n\t\tif err != nil {\n\t\t\treturn err\n\t\t}\n\t\t_, err = hcp.NewHvnRoute(ctx, \"main-to-dev\", \u0026hcp.HvnRouteArgs{\n\t\t\tHvnLink:         main.SelfLink,\n\t\t\tHvnRouteId:      pulumi.String(\"main-to-dev\"),\n\t\t\tDestinationCidr: pulumi.String(\"172.31.0.0/16\"),\n\t\t\tTargetLink:      dev.SelfLink,\n\t\t})\n\t\tif err != nil {\n\t\t\treturn err\n\t\t}\n\t\t_, err = ec2.NewVpcPeeringConnectionAccepter(ctx, \"peerVpcPeeringConnectionAccepter\", \u0026ec2.VpcPeeringConnectionAccepterArgs{\n\t\t\tVpcPeeringConnectionId: dev.ProviderPeeringId,\n\t\t\tAutoAccept:             pulumi.Bool(true),\n\t\t})\n\t\tif err != nil {\n\t\t\treturn err\n\t\t}\n\t\treturn nil\n\t})\n}\n```\n{{% /example %}}\n{{% /examples %}}\n\n## Import\n\n# The import ID is {hvn_id}:{peering_id}\n\n```sh\n $ pulumi import hcp:index/awsNetworkPeering:AwsNetworkPeering peer main-hvn:11eb60b3-d4ec-5eed-aacc-0242ac120015\n```\n\n ",
```

Comment:
i'm assuming this is all generated? if so, I have a comment later on about generated code that this may be worth applying to

Reply:
Yes, this is generated; I'll see about adding it to .gitattributes.